src/App/components/JsPsychExperiment.jsx

```diff
@@ -51,7 +51,7 @@ export default function JsPsychExperiment({
       tempJsPsych.data.addProperties({
         app_name: import.meta.env.PACKAGE_NAME,
         app_version: import.meta.env.PACKAGE_VERSION,
-        app_commit: await window.electronAPI.getCommit(),
+        // app_commit: await window.electronAPI.getCommit(),
```

Comment:
commented this out because I wasn't sure why it errored with reading as undefined when running in browser

Reply:
@YUUU23 What was the error you were getting?

Reply:
Just re-ran honeycomb with different settings and it seems like this error went away! Not sure if it was a previous dependency issue or something but seems like everything is ok now!